test/user/model.js

```diff
@@ -4,7 +4,12 @@
 var should = require('should'),
     app = require('../../server'),
     mongoose = require('mongoose'),
+<<<<<<< HEAD:test/user/model.js
```

Comment:
@ibrahim013 merge conflicts, did you rebase off staging

Reply:
nope